doc/tutorials/04-lattice_boltzmann/04-lattice_boltzmann_part1.ipynb

```diff
@@ -8,7 +8,7 @@
     "\n",
```

Comment:
<p>you introduced a typo</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/KaiSzuttor/espresso/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>